src/Dialect/Krnl/Krnl.td

```diff
@@ -12,7 +12,7 @@
 // When you perform a change to the KRNL dialect, please follow the steps below
 // to update the Krnl dialect documentation.
 //
-// 1) cd onnx-mlir/build 
+// 1) cd onnx-mlir/build
```

Comment:
I am guilty of the same all the time, so this is a memo to myself too. In the future, it is better to have  non-functional changes like spelling/spaces updates in a separate PR: one for non-functional changes that folks can approve very quickly, and one for the functional changes where reviewers pay more attentions to the actual changes.

Reply:
Ohh this was by accident...my visual studio code fixes certain white spaces and indentation once I save the file. Sorry about that :)

Reply:
But here the whitespace was removed on line 15....so I think we should be fine